File: PaymentSDK/UIHostObjCApp/models/OrderRequestPayload.m
//
//  OrderRequestPayload.m
//  UIHostObjCApp
//
//  Created by Rahul Dhuri on 04/06/19.
//  Copyright © 2019 Network International. All rights reserved.
//

#import "OrderRequestPayload.h"

@implementation OrderRequestPayload

+(instancetype)initWith:(Amount *)amount action:(NSString *)action language:(NSString *)language andDescription:(NSString *)description {
    OrderRequestPayload *payload = [[OrderRequestPayload alloc] init];
    payload.amount = amount;
    payload.action = action;
    payload.language = language;
    payload.productDescription = description;
    return payload;
}


@end

File: PaymentSDK/UIHostApp/models/OrderRequestPayload.swift
//
//  OrderRequestPayload.swift
//  merchant-sample-app
//
//  Created by Niraj Chauhan on 4/26/19.
//  Copyright © 2019 Niraj Chauhan. All rights reserved.
//

import Foundation

struct OrderRequestPayload : Codable {
    var amount: Amount
    var action : String
    var language : String
    var description : String
}
